pkg/controller/migmigration/postValidate.go

```diff
@@ -0,0 +1,298 @@
+package migmigration
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// EventDisplay is a structure for displaying occured events
+type EventDisplay struct {
+	Pod           string    `json:"pod"`
+	LastTimestamp time.Time `json:"lastTimestamp,omitempty"`
+	Message       string    `json:"message,omitempty"`
+	Reason        string    `json:"reason,omitempty"`
+}
+
+const (
+	// ValidationFailed is a failed validation state
+	ValidationFailed = iota
```

Comment:
The term `Validation` has a pretty specific meaning in our controller/CRs.  I pertains to assessing the stage of the world prior to running a migration.  Suggest something like `Verification` for the _post_ checks would be clearer.

That said, I think you can replace these return values by having the task code chec the `Reason = (Running|Succeeded|Failed)` in the `PostVerification` (or some other name) condition.  Will describe condition changes elsewhere.

Reply:
Thanks for pointing it out. Going to replace the naming, sorry for confusion.